thunder/executors/torch_compile.py

```diff
@@ -75,23 +79,23 @@ def torch_interpreted_func(*args):
     # TODO: issue "Try using _transform_for_operator_executor_execution for
     # torch.compile executor"
     torch_trace = trace(inline_trace=False)(torch_interpreted_func, *sorted_unique_inputs)
-    compiled_func = torch.compile(torch_trace.python_callable())
-
+    compiled_func = torch.compile(torch_trace.python_callable(), fullgraph=True)
```

Comment:
Would this work equally?
```suggestion
    compiled_func = torch.compile(torch_trace.python_callable(include_decorators=False), fullgraph=True)
```

Reply:
I would expect that it does as long as there is an outer trace with the decorator.

Is having the same decorators in every trace the intended design? I thought so, as doing so means that the traces are independent: where you execute it doesn't change its result.

But if reviewers prefer to drop the decorator here, I could try it

Reply:
"no_autocast" just looks nicer when we print traces.